aten/src/THC/generic/THCTensorIndex.cu

```diff
@@ -4,6 +4,14 @@
 
 #include "ATen/cuda/CUDAContext.h"
 
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
let's name them INDEX_BLOCK_SIZE and INDEX_GRID_DIM to clarify intent.

Reply:
Sure.